static/js/gtag.js

```diff
@@ -7,9 +7,18 @@ gtag('config', 'UA-26406144-13');
 // with a 'data-ga-click-event' attribute is clicked.
 window.addEventListener('DOMContentLoaded', function () {
   function sendEvent(e) {
-    gtag('event', e.currentTarget.dataset.gaClickEvent, {
+    var elem = e.currentTarget;
+    var callback;
+    if (elem.dataset.gaClickHref) {
```

Comment:
Or ask if `e.target.attributes[href] != null` or something like that... Then we don't need additional data properties...

Otherwise, I suggest we make so you only have one data property, but we have two different ones depending on how send the metric..

Reply:
> Or ask if e.target.attributes[href] != null or something like that... Then we don't need additional data properties...

I've started with this, but the template logic can be complex, e.g. in case of the analysis/score tab, and neither the `.target` nor the `.currentTarget` has the `<a href=""` reference. I think there is a benefit of adding an explicit attribute for it.

Reply:
We could also put the `data-ga-click-event` property on the `<a>` tag...

Then refactor the template to include the `<a>` tag... I know it means more code, but injecting HTML into our template as we currently do is undesirable..

Reply:
We have four kinds of tabs now:
- `li` without a link, text content (on readme tab, linking to changelog)
- `li` without a link, complex content (on readme tab, linking to score)
- `li` with a link, text content (on versions tab, linking to changelog)
- `li` with a link, complex content (on versions tab, linking to score)

We could add `data-ga-click-event` to `a` links inside the tabs, but then they the `li` element should not have it, otherwise it will be counted twice. The tab template will be very similar, different attribute will be masked conditionally. I'm fine either way, just wanted to make it clear before I do the change.